gaia-fast-list.js

```diff
@@ -138,6 +180,11 @@ var GaiaFastListProto = {
         background: var(--background);
         -moz-user-select: none;
         text-decoration: none;
+        will-change: initial !important;
+      }
+
+      :host(.layerize) ::content .gfl-item {
+        will-change: transform !important;
```

Comment:
Only apply costly `will-change` property when the component had the `.layerize` class.